qcodes/station.py

```diff
@@ -333,7 +334,7 @@ def load_config_file(self, filename: Optional[str] = None) -> None:
             self.load_config(f)
 
     def load_config_files(self,
-                          *filenames: List[Union[str, Path]]
```

Comment:
Why did this change from using `*filenames` to not. We do want to use `*` args here

Reply:
Yes, I am working on it. Tests were failing with `*` and I am figuring out why.

Reply:
Ok, I would suggest that you work on that by running the tests locally before committing and pushing

Reply:
Ok, will do that going forward.